### PR TITLE
User chat mentions

### PR DESCRIPTION
**Pull request by poixen**
**This is ready to be tested.**

When other users are mentioned with @thiername you see it highlighted. The name is also href'd to thier account, so you can click to add a mention, or right click to interact with the user.

other user mentions

### CODE REVIEW

File: cockatrice/src/chatview.cpp [hunk 1]
```diff
@@ -6,10 +6,15 @@
...
-#include "tab_supervisor.h"
 #include "pixmapgenerator.h"
 #include "settingscache.h"
 #include "main.h"
+#include "userlist.h"
+#include "tab_userlists.h"
+
+const QColor OTHER_USER_MENTION_COLOR = QColor(145, 210, 255); // light blue
+const QColor MENTION_COLOR = QColor(190, 25, 85); // maroon
+const QColor OTHER_USER_COLOR = QColor(0, 65, 255); // dark blue
```
Comment (Daenyth) [inline cockatrice/src/chatview.cpp]: :+1:

**Comment by ZeldaZach** [comment]
Good to go :+1:

**Comment by Daenyth** [comment]
Minor ui request: Is it possible to add some border radius to the color so it's not so sharp? A rounded corner would make it look quite a bit nicer. I spent a little time checking the Qt docs and couldn't find something obvious, but perhaps you're more familiar than I am.

Barring that one thing, this is ready to merge :+1:

**Comment by tooomm** [comment]
Are users able to configure both their own and mentions of other users with the upcoming color wheel(s)?

I think right now the "other user mentions" are a bit too stressed after some testing even with the soft color. It still stands out very much. Not sure if there is some more subtle way of highlighting them.
But I like them to be there generally, just prefer e.g. a very very light blue background with no change in boldness or color to their name. Or maybe just color + boldness without background...

**Comment by Daenyth** [comment]
Bold + clickable alone may be enough

**Pull request merged**